electron/src/lib/openGraph.ts

```diff
@@ -200,7 +200,7 @@ export const getOpenGraphDataAsync = async (url: string): Promise<OpenGraphResul
     metadata.image = metadata.image[0];
   }
 
-  if (typeof metadata.image === 'object' && !Array.isArray(metadata.image) && metadata.image.url) {
+  if (typeof metadata.image === 'object' && metadata.image?.url) {
```

Comment:
If I see that correctly `metadata.image` can be an object or an array. if it's an array typeof === 'object' will return true. So in case `metadata.image` is an array this will break imho.

Reply:
AFAIC this will not break since JS will realize that `metadata.image.url` doesn't exist.

Reply:
Mh, you are right